posthog/cdp/templates/aws_kinesis/template_aws_kinesis.py

```diff
@@ -108,9 +108,9 @@
             "default": "us-east-1",
         },
         {
-            "key": "aws_kinesis_stream_arn",
+            "key": "aws_kinesis_stream_name",
             "type": "string",
-            "label": "Kinesis Stream ARN",
+            "label": "Kinesis Stream Name",
```

Comment:
```suggestion
            "label": "Kinesis Stream Name (ARN)",
```
might be a good compromise

Reply:
I think asking for the ARN is straightforwardly incorrect here, and [stream names are a thing with a spec in kinesis land](https://docs.aws.amazon.com/kinesis/latest/APIReference/API_CreateStream.html#API_CreateStream_RequestSyntax), so it's better to be precise